api/internal/loader/fileloader_test.go

```diff
@@ -659,6 +660,35 @@ func TestLoaderHTTP(t *testing.T) {
 		_, err := l2.Load(x.path)
 		require.Error(err)
 	}
+
+	var testCaseRedirect = []testData{
+		{
+			path:            "https://example.com/resource.yaml",
+			expectedContent: "https content",
+		},
+	}
+	for _, x := range testCaseRedirect {
+		expectedLocation := "https://redirect.com/resource.yaml"
+		hc := makeFakeHTTPClient(func(req *http.Request) *http.Response {
+			response := &http.Response{
+				StatusCode: 300,
+				Body:       io.NopCloser(bytes.NewBufferString("")),
+				Header:     make(http.Header),
+			}
+			response.Header.Add("Location", expectedLocation)
+			return response
+		})
+		l2 := l1
```

Comment:
What does `l1` and `l2` mean?

Reply:
It is loaders. I use solution from testcase from https://github.com/kubernetes-sigs/kustomize/blob/c1de0301f5e71ffbecf77f5bc8687e7693878eb5/api/internal/loader/fileloader_test.go#L638